src/extension.ts

```diff
@@ -6,16 +6,12 @@ const extName = 'ocm-vscode-extension';
 const cmdNewProjectName = 'ocmNewProject';
 
 // DISPOSABLES
-function cmdNewProjectDisposable (): vscode.Disposable {
-	return vscode.commands.registerCommand(
-		`${extName}.${cmdNewProjectName}`, async () => {
-			await newProject.create();
-		}
-	);
-}
+const cmdNewProjectDisposable = vscode.commands.registerCommand(
+	`${extName}.${cmdNewProjectName}`, () => newProject.create()
+);
 
 // EXPORTS
-export async function activate(context: vscode.ExtensionContext) {
+export function activate(context: vscode.ExtensionContext) {
```

Comment:
why did you remove the async ?

Reply:
It's not required at that point anymore.
it was a leftover from some older iteration when the disposable came for an async function that needed awaiting.